TimedArithmeticQuiz/src/utility/newId.js

```diff
@@ -0,0 +1,6 @@
+let lastId = 0;
+
+export default function(prefix='id') {
+    lastId++;
+    return `${prefix}-${lastId}`;
+}
```

Comment:
do named export here

Reply:
disable next button if no input has provided

Reply:
done both the points